package.json

```diff
@@ -8,6 +8,9 @@
   },
   "scripts": {
     "babel-node": "babel-node --presets=env",
+    "migrate-undo": "sequelize db:migrate:undo:all",
+    "migrate": "npm run migrate-undo && sequelize db:migrate",
+    "reset:db": "sequelize db:migrate:undo:all && sequelize db:migrate",
     "migrate-reset": "sequelize db:migrate:undo:all && sequelize db:migrate",
```

Comment:
migrate-reset and reset:db are doing the same thing. I don't see the need for reset:db

Reply:
done